capabilities/capabilities_test.go

```diff
@@ -0,0 +1,72 @@
+package capabilities
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+type testInput struct {
+	Provider       string
+	ReleaseVersion string
+}
+
+var capabilityTests = []struct {
+	in  testInput
+	out []CapabilityDefinition
+}{
```

Comment:
Maybe add one or two tests with `ReleaseVersion` that doesn't parse (to verify that error is returned).

Reply:
Done